rvv-intrinsic-rfc.md

```diff
@@ -254,18 +254,10 @@ vuint8m1_t vadd_vx_u8m1(vuint8m1_t op1, uint8_t op2, size_t vl);
 vuint64m1_t vadd_vx_u64m1(vuint64m1_t op1, uint64_t op2, size_t vl);
 ```
 
-The compiler may generate multiple instructions for the intrinsics. For example, it is a little bit complicated to support `uint64_t` operations under `XLEN` = 32.
-
-It breaks the one-to-one mapping between intrinsics and assembly mnemonics in some hardware configurations. However, it makes more sense for users to use the scalar types consistent with the `SEW` of vector types.
-
-There is the same issue for `vmv.x.s`, `vmv.s.x`, `vfmv.f.s`, `vfmv.s.f`, `vslide1up.vx`, `vfslide1up.vf`, `vslide1down.vx`, and `vfslide1down.vx`. Use `SEW` to encode the scalar type.
-
-```
-Example:
-
-// Use uint8_t for op2.
-vuint8m1_t vslide1up_vx_u8m1(vuint8m1_t dest, vuint8m1_t op1, uint8_1 op2);
-```
+To support the implied semantics, the compiler may generate multiple instructions for the intrinsics.
+(More generally, this API does not constrain the compiler in which instructions it actually generates.)
+For example, to support `vmv_x_s_i64m1_i64` when XLEN = 32, the compiler may generate `vmv.x.s`, `vslide1down.vx`, and another `vmv.x.s`, all with SEW = 32, to extract the full 64 bits into two X-registers.
+(A future API extension could include an "`vmv_x_s_i64m1_i32`", perhaps only when XLEN = 32, whose semantics match `vmv.x.s` for XLEN = 32 and SEW = 64.)
```

Comment:
I think it will be better if we track this under `Issue`, or else this idea may get lost in this rfc file.